packages/@uppy/companion/src/standalone/index.js

```diff
@@ -120,7 +120,8 @@ module.exports = function server (inputCompanionOptions = {}) {
   if (companionOptions.redisUrl) {
     const RedisStore = connectRedis(session)
     const redisClient = redis.client(companionOptions)
-    sessionOptions.store = new RedisStore({ client: redisClient })
+    // todo next major: change default prefix to something like "companion:"
+    sessionOptions.store = new RedisStore({ client: redisClient, prefix: process.env.COMPANION_REDIS_EXPRESS_SESSION_PREFIX || 'sess:' })
```

Comment:
What about making the default prefix include `companion`? Other people running companion may also get confused what these `sess:` things are in their redis without such clues?

Reply:
I would also prefer that, thrn we might not even need an option. But I think that would be a breaking change and not sure if we have a major of companion coming up

Reply:
We have a place for storing Major suggestions right? then I propos we keep this now, but add the reminder there for when the next major does come up?

Reply:
I think we usually store them as TODOs in the code (there is already a todo there to change the default)